src/taskpane/excel.ts

```diff
@@ -9,10 +9,14 @@ import { getGraphData } from "./../helpers/ssoauthhelper";
 
 Office.onReady((info) => {
   if (info.host === Office.HostType.Excel) {
-    document.getElementById("getProfileButton").onclick = getGraphData;
+    document.getElementById("getProfileButton").onclick = run;
```

Comment:
I don't like changing `getGraphData` to `run`. Having a more descriptive name is better!

Reply:
The 'run' is consistent with the other templates in which the run call in this same file is wired up and all the action happens there.  On top of that, with the changes made in the PR the getGraphData call needs to have an argument.